src/main/features/claim/routes/pay.ts

```diff
@@ -141,8 +147,21 @@ export default express.Router()
       draft.document.claimant.payment = payment
       await new DraftService().save(draft, user.bearerToken)
 
+      trackCustomEvent(`Pre payment for externalId: ${externalId}`,{
```

Comment:
I dont think there's any value in tracking this event, we only really need to know when this fails.

Reply:
The reason why I originally put them in is to ensure that the microservices being used in pay.ts were behaving. Also, I also wanted to make sure that the RC emails being generated reflected the correct status. However, for the time being I see your point and have removed them